dev-lisp/clisp/clisp-2.49.93~git.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="ANSI Common Lisp Implementation"
+DESCRIPTION="GNU CLISP is an ANSI Common Lisp Implementation. \
+It is known for high portability, small image size and efficient bignums."
+HOMEPAGE="https://clisp.sourceforge.io/"
+COPYRIGHT="1992-1993 Bruno Haible, Michael Stoll
+	1994-1997 Bruno Haible, Marcus Daniels
+	1998 Bruno Haible, Pierpaolo Bernardi, Sam Steingold
+	1999-2000 Bruno Haible, Sam Steingold
+	2001-2018 Sam Steingold, Bruno Haible
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="c735dd548ba2365804bcee9c2bdd3c291657e8ae"
+SOURCE_URI="https://gitlab.com/gnu-clisp/clisp/-/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="7c29554297e5d27abd7e50fff7691b4dc2ee26bd694369b238fb368b38278bf7"
+SOURCE_DIR="clisp-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	clisp$secondaryArchSuffix = $portVersion
+	cmd:clisp$secondaryArchSuffix = $portVersion
+	cmd:clisp_link$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsigsegv$secondaryArchSuffix
+	devel:libffcall$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix >= 8
+	devel:libintl$secondaryArchSuffix
+	devel:libunistring$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:cmp
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt
+	"
+
+BUILD()
+{
+	./configure \
+		FORCE_UNSAFE_CONFIGURE=1 \
+		LDFLAGS="-lbsd -lnetwork" \
+		--with-included-regex \
+		--prefix=$prefix \
+		--datarootdir=$dataDir \
+		--bindir=$binDir \
+		--libdir=$libDir \
+		--docdir=$docDir \
+		--mandir=$manDir \
+		build-dir
```

Comment:
Would it make this shorter if `runConfigure` is used instead of specifying all the variables manually? Some of them would need to be omitted then (`binDir` for sure, see the comment above from @Begasus).

Reply:
@jmairboeck  Tried that from the beginning (runConfigure), it failed with datarootdir/sysconfigdir and others, this is easier the omitting all of those :)